cluster/addons/metrics-server/metrics-server-deployment.yaml

```diff
@@ -60,10 +59,15 @@ spec:
         - --kubelet-port=10255
         - --deprecated-kubelet-completely-insecure=true
         - --kubelet-preferred-address-types=InternalIP,Hostname,InternalDNS,ExternalDNS,ExternalIP
+        - --cert-dir=/tmp
+        - --secure-port=4443
         ports:
-        - containerPort: 443
+        - containerPort: 4443
```

Comment:
should there be a corresponding service change mapping 443 to 4443?

Reply:
Service specifies "https" in target port. If I remember correctly this should work, but would be good to test. @x13n

Reply:
yes, that will work

Reply:
the routing will work but can still have implications for some network setups... would prefer to keep this on 443

Reply:
/hold for port revert

Reply:
Updated the image to 0.4.4 that works with port 443, but it not released yet. Need to wait for https://github.com/kubernetes-sigs/metrics-server/issues/731 to close.